File: benchmarks/elixir/phoenix-ecto-postgres/lib/wfb_phoenix/models/post.ex
defmodule WfbPhoenix.Models.Post do
  use Ecto.Schema

  schema "posts" do
    field :title, :string
    field :content, :string
    field :views, :integer
    field :created_at, :naive_datetime

    belongs_to :user, WfbPhoenix.Models.User
  end
end

File: benchmarks/ruby/rails/app/models/post.rb
class Post < ApplicationRecord
  self.table_name = 'posts'
  belongs_to :user

  def to_h
    {
      id: id,
      title: title,
      content: content,
      views: views,
      createdAt: created_at.strftime('%Y-%m-%dT%H:%M:%SZ')
    }
  end
end
